gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx

```diff
@@ -0,0 +1,53 @@
+---
+date: '2023-02-28'
+title: Synapse 1.78 released
+categories:
+- Releases
+author: H. Shay
+---
+
+Greetings Matrix fans! We've published Synapse [version 1.78](https://github.com/matrix-org/synapse/releases/tag/v1.78.0)
+as the new stable release this week. Synapse admins are encouraged to upgrade
+to it at their convenience. Please take a look at the [upgrade notes](https://matrix-org.github.io/synapse/v1.78/upgrade.html#upgrading-to-v1780)
+for any important information about upgrading.
+
+## Announcements
+
+Please note that Synapse 1.78.0 replaces the `/_synapse/admin/v1/media/<server_name>/delete` admin API with an identical
+endpoint at `/_synapse/admin/v1/media/delete`. Please update your tooling to use the new endpoint. The deprecated version
+will be removed in a future release.
+
+## The new stuff
+
+In case you were unaware, Synapse has a command line export data command which allows administrators to export data for
+a specific user (you can read more about this command [here](https://matrix-org.github.io/synapse/latest/usage/administration/admin_faq.html#how-can-i-export-user-data)).
+Synapse 1.78.0 updates this command adding account data to the user information that is returned by the export data command.
+
+This version of Synapse also features a few changes to push rules, with implementations for [MSC3758](https://github.com/matrix-org/matrix-spec-proposals/pull/3758),
+[MSC3966](https://github.com/matrix-org/matrix-spec-proposals/pull/3966), and the removal of the spurious `dont_notify`
+action from the defaults for the `.m.rule.reaction` pushrule.
+
+Synapse v1.78.0 includes some fixes for faster-joins related bugs as we work out the kinks. Some of these include a fix
+for a bug introduced in Synapse 1.76.0 where partially-joined rooms could not be deleted using the
+[purge room API](https://matrix-org.github.io/synapse/latest/admin_api/rooms.html#delete-room-api), and
+a fix for a bug introduced in Synapse 1.75 where the [portdb script](https://matrix-org.github.io/synapse/latest/postgres.html#porting-from-sqlite)
+would fail to run after a room had been faster-joined. We continue to work to polish faster joins, and thank everyone who
+filed an issue.
+
```

Comment:
I would be tempted to add an honorary mention https://github.com/matrix-org/synapse/issues/15150, which is a bug that would be biting workerised deployments.

Reply:
good shout will do